src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/build.py

```diff
@@ -260,10 +263,14 @@ def acr_build(cmd,
               secret_build_arg=None,
               docker_file_path='Dockerfile',
               no_push=False,
-              no_logs=False):
+              no_logs=False,
+              os_type=None):
     _, resource_group_name = validate_managed_registry(
         cmd.cli_ctx, registry_name, resource_group_name, BUILD_NOT_SUPPORTED)
 
+    if os_type and os_type.lower() not in SUPPORTED_OS_TYPES:
+        raise CLIError("The os type is invalid. Supported values are '{!s}'".format(SUPPORTED_OS_TYPES))
```

Comment:
Not needed, the CLI arg parser already handles this.

Reply:
Yes, please remove.